crates/wasmcloud-host/src/lib.rs

```diff
@@ -1,29 +1,63 @@
+//! in using a "stock" binary to run your actor workloads and communicate with capability providers using standard
+//! features, then you should use the [wasmCloud](https://wasmcloud.dev/overview/installation/) binary available for installation.
+//!
+//! If, on the other hand, you are interested in providing a custom host runtime of your own that utilizes the wasmCloud
+//! host API as a platform, then this crate is what you'll need.
 //!
-//! By default a wasmCloud host will start in offline mode and only allow "local" scheduling of actors
-//! and capabilities. If you choose to opt-in to the lattice, you can use NATS as a message broker to
-//! provide the infrastructure for wasmCloud's self-forming, self-healing network. If you then want
-//! even more power, you can choose to override the capability provider used for managing the shared
-//! state within a lattice.
+//! To start a runtime, simply build a host and then add actors, capabilities, and link definitions to it.
+//! For more information, take a look at the documentation and tutorials at [wasmcloud.dev](https://wasmcloud.dev).
 //!
-//! Local development on your workstation is easy and simple by default, and you should only
-//! incur additional complexity as you move toward resilient, distributed production
-//! environments.
+//! # Host API Example
 //!
-//! To start a runtime, simply add actors and capabilities to the host. For more information,
-//! take a look at the documentation and tutorials at [wasmcloud.dev](https://wasmcloud.dev).
+//! The following example creates a new wasmCloud host in the default standalone (no lattice/single-player) mode. It
+//! then loads an actor that echoes back incoming HTTP requests as a JSON object in the body of the outbound HTTP response.
 //!
-//! # Example
-//! The following example creates a new wasmCloud host in the default standalone (no lattice) mode. It
-//! then loads an actor that simply echoes back incoming HTTP requests as outbound HTTP responses.
-//! The HTTP server capability provider is loaded so that the actor can receive web requests.
-//! Note that the link definition (configuration of the link between the actor and the
-//! capability provider) can be defined _in any order_. The host runtime automatically
-//! establishes links as soon as all related parties are up and running inside a host or
-//! a lattice.
+//! The HTTP server capability provider is loaded so that the actor can receive web requests. A [link definition](https://wasmcloud.dev/reference/host-runtime/links/) is required
+//! between the HTTP server capability provider and the actor in order to verify actor privileges and supply configuration values
+//! (such as the port on which to listen). This link definition can be established _before or after_ the actor and capability
+//! provider have been started, as link definitions are first-class data cached throughout a lattice.
```

Comment:
Thoughts on linking the `lattice` keyword here to https://wasmcloud.dev/reference/lattice/?

Reply:
I like it.